.gitignore

```diff
@@ -1 +1,37 @@
-.vscode/
\ No newline at end of file
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+.vscode/
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# next.js
+/.next/
+/out/
+
+# production
+/build
```

Comment:
This may affect current deployment procedures because build will not be captured by git

Reply:
If build is being updated with the intent for deploy, the deployment isn't going to get updated

Reply:
thanks, Ill remove it just in case

Reply:
I will not be using a flow that assumes build folder but Ill remove for now

Reply:
also delete build from your computer, commit, rebuild, commit